src/quote/swap/adapters/uniswap/index.ts

```diff
@@ -12,7 +15,7 @@ import { Exchange, isSameAddress } from 'utils'
 
 function changeToWethIfNecessary(token: string, chainId: number): string {
   if (isSameAddress(token, EthAddress)) {
-    const weth = getTokenData('WETH', chainId)
+    const weth = getTokenByChainAndSymbol(chainId, 'WETH')
```

Comment:
Here you can just return `getTokenByChainAndSymbol(chainId, 'WETH')` because WETH is present on every network we use, so it will for sure return a token.